### DISCUSSION TITLE
ITS Black Friday!!!

### DISCUSSION THREAD

**Discussion by gerardopenya** [aclap-dev/video-downloadhelper#1503] [License & Purchase requests]
### What's your issue?

No issue... just that... is Black Friday... no discount in licence?  :) 

### I confirm that I did not not paste my full license number

- [X] I confirm

**Comment by jcfrog** [comment]
Hi,
No, sorry, there is no discount.

**Reply by wbui567** [reply]
hello, why you increase the license for firefox, before, it was 18.5 euros/usd, now it's 28.5, really too expensive!

**Reply by jcfrog** [reply]
Nothing has changed for years. It has never been $18.50.

**Reply by wbui567** [reply]
ah, yes, I made confusion with license for chrome.
But, I think, If you do a cyber mon day price or a black friday price, clearly, I'll buy a license.
